ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/TPraos.hs

```diff
@@ -253,7 +255,7 @@ newtype TPraosHorizonView = TPraosHorizonView {
     }
 
 newtype instance Ticked TPraosHorizonView = TickedTPraosHorizonView {
-      tickedTpraosHorizonViewExtraEntropy :: SL.Nonce
+      tickedTPraosHorizonViewExtraEntropy :: SL.Nonce
```

Comment:
Minor: preferably this diff is squashed into the commit that introduces `TickedTPraosHorizonView`